### DISCUSSION TITLE
Shairport-sync - AirPlay rendering device

### DISCUSSION THREAD

**Discussion by BV-Labs** [rern/rAudio#99] [Fixed Issues]
# systemctl status shairport-sync

● shairport-sync.service - Shairport Sync - AirPlay Audio Receiver
     Loaded: loaded (/usr/lib/systemd/system/shairport-sync.service; enabled; vendor preset: disabled)
    Drop-In: /etc/systemd/system/shairport-sync.service.d
             └─override.conf
     Active: **failed** (Result: exit-code) since Sun 2021-01-10 12:41:47 EET; 3s ago
    Process: 3000 ExecStart=/usr/bin/taskset -c 3 /usr/bin/shairport-sync (code=exited, status=1/FAILURE)
   Main PID: 3000 (code=exited, status=1/FAILURE)

Jan 10 12:41:47 rAudio systemd[1]: Started Shairport Sync - AirPlay Audio Receiver.
Jan 10 12:41:47 rAudio shairport-sync[3000]: fatal error: Line 8 of the configuration file "/etc/shairport-sync.conf":
                                             syntax error
Jan 10 12:41:47 rAudio systemd[1]: shairport-sync.service: Main process exited, code=exited, status=1/FAILURE
Jan 10 12:41:47 rAudio systemd[1]: shairport-sync.service: Failed with result 'exit-code'.

**Comment by BV-Labs** [comment]
general = {
        name = "rAudio"
}
sessioncontrol = {
        run_this_before_play_begins = "/usr/bin/sudo /srv/http/bash/shairport.sh";
        run_this_after_play_ends = "/usr/bin/sudo /srv/http/bash/shairport.sh stop";
}
^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@^@alsa = {
        output_device = "hw:0";
        mixer_control_name = "( not available )";
}
alsa = {
        output_device = "hw:0";
        mixer_control_name = "( not available )";
}

**Comment by rern** [comment]
Try
```sh
cat << EOF > /etc/shairport-sync.conf
general = {
	name = "rAudio"
}
sessioncontrol = {
	run_this_before_play_begins = "/usr/bin/sudo /srv/http/bash/shairport.sh";
	run_this_after_play_ends = "/usr/bin/sudo /srv/http/bash/shairport.sh stop";
}
alsa = {
	output_device = "hw:0";
}
EOF
```
The enable again

**Comment by BV-Labs** [comment]
Well done!
